go/chat/inboxsource.go

```diff
@@ -845,6 +849,20 @@ func (s *HybridInboxSource) Search(ctx context.Context, uid gregor1.UID, query s
 	return res, nil
 }
 
+func (s *HybridInboxSource) IsTeam(ctx context.Context, uid gregor1.UID, item string) (res bool, err error) {
+	defer s.Trace(ctx, func() error { return err }, "IsTeam")()
+	_, convs, err := s.createInbox().ReadAll(ctx, uid, true)
```

Comment:
why not use `Read` with `MembersTypes`  set? otherwise this bypasses the query caching right?

Reply:
No this just gives whatever we have in cache back.